src/tasks/WasmAppBuilder/PInvokeTableGenerator.cs

```diff
@@ -137,7 +132,7 @@ private void EmitPInvokeTable(StreamWriter w, Dictionary<string, string> modules
                 Where(l => l.Module == module && !l.Skip).
                 OrderBy(l => l.EntryPoint).
                 GroupBy(d => d.EntryPoint).
-                Select(l => "{\"" + _fixupSymbolName(l.Key) + "\", " + _fixupSymbolName(l.Key) + "}, " +
+                Select(l => "{\"" + l.Key + "\", " + TransformEntryPoint(l.First()) + "}, " +
```

Comment:
Can we use `l.Key` instead of `TransformEntryPoint(l.First())`? It seems to me that it's the same

Reply:
no, the l.key is unmangled here (I'll merge an escaping fix for c literals shortly). TransformEntryPoint(l.First()) will get you the c symbol. They have to be able to be different.

Reply:
Added calls to escape the literal string in case the one of the names was something like `mymod"tacobell`